test/e2e/specs/onboarding/ftme__site-assembler.ts

```diff
@@ -83,43 +83,31 @@ describe( 'Site Assembler', () => {
 	} );
 
 	describe( 'Site Assembler', function () {
-		let assembledPreviewLocator: Locator;
-		let startSiteFlow: SiteAssemblerFlow;
+		let siteAssemblerFlow: SiteAssemblerFlow;
 
 		beforeAll( async function () {
-			startSiteFlow = new SiteAssemblerFlow( page );
-			assembledPreviewLocator = page.locator( '.pattern-large-preview__patterns .block-renderer' );
+			siteAssemblerFlow = new SiteAssemblerFlow( page );
 		} );
 
 		it( 'Select "Header"', async function () {
-			await startSiteFlow.selectLayoutComponentType( 'Header' );
-			await startSiteFlow.selectLayoutComponent( 1 );
+			await siteAssemblerFlow.selectLayoutComponentType( 'Header' );
+			await siteAssemblerFlow.selectLayoutComponent( 'Simple Header' );
 
-			expect( await assembledPreviewLocator.count() ).toBe( 1 );
+			expect( await siteAssemblerFlow.getAssembledComponentsCount() ).toBe( 1 );
 		} );
 
 		it( 'Select "Sections"', async function () {
-			await startSiteFlow.selectLayoutComponentType( 'Sections' );
-			await startSiteFlow.selectLayoutComponent( 1 );
+			await siteAssemblerFlow.selectLayoutComponentType( 'Sections' );
+			await siteAssemblerFlow.selectLayoutComponent( 'Heading with two images and descriptions' );
 
-			expect( await assembledPreviewLocator.count() ).toBe( 2 );
+			expect( await siteAssemblerFlow.getAssembledComponentsCount() ).toBe( 2 );
 		} );
 
 		it( 'Select "Footer"', async function () {
-			await startSiteFlow.selectLayoutComponentType( 'Footer' );
-			await startSiteFlow.selectLayoutComponent( 1 );
+			await siteAssemblerFlow.selectLayoutComponentType( 'Footer' );
+			await siteAssemblerFlow.selectLayoutComponent( 'Simple centered footer' );
 
-			expect( await assembledPreviewLocator.count() ).toBe( 3 );
-		} );
-
-		it( 'Click "Save and continue" and land on the Site Editor', async function () {
```

Comment:
@worldomonation I agree that loading the site editor here probably doesn't have value, but should we check to ensure the 'save and continue' function works by clicking the button and verifying no errors are thrown? Or should we suggest to the dev that worked on this spec to cover that outside of E2E?

Reply:
@youbrokesomething Yes, good catch - I had removed this portion while I was stress testing and forgot to add it back in. Thanks for noticing!

Reply:
Also, according to the folks at Team Lego there is value in actually saving then checking the editor loads.
p1692260392212649-slack-CRWCHQGUB